services/sts/src/main/java/software/amazon/awssdk/services/sts/auth/StsAssumeRoleWithWebIdentityCredentialsProvider.java

```diff
@@ -92,6 +95,11 @@ private Builder() {
          * @return This object for chained calls.
          */
         public Builder refreshRequest(AssumeRoleWithWebIdentityRequest assumeRoleWithWebIdentityRequest) {
+            this.assumeRoleWithWebIdentityRequest = () -> assumeRoleWithWebIdentityRequest;
+            return this;
+        }
+
+        public Builder refreshRequest(Supplier<AssumeRoleWithWebIdentityRequest> assumeRoleWithWebIdentityRequest) {
             this.assumeRoleWithWebIdentityRequest = assumeRoleWithWebIdentityRequest;
```

Comment:
Is this the same pattern as the other providers?
+ Javadoc.

Reply:
Mostly. Other provider passes the non-supplier to the supplier method instead of directly setting. Will update.